packages/salesforcedx-vscode-core/src/context/workspaceContext.ts

```diff
@@ -43,6 +44,11 @@ export class WorkspaceContext {
     return await WorkspaceContextUtil.getInstance().getConnection();
   }
 
+  public async getSfdxNamespace(): Promise<string | undefined> {
+    const namespace = await SfdxProjectConfig.getValue('namespace');
+    return typeof namespace === 'string' ? namespace : undefined;
+  }
```

Comment:
@gbockus-sf Yeah, I've considered adding another exported API vs overloading the workspaceContext. The sfdx namespace exists in the `sfdx-project.json` file, which is located in the workspace, so I thought it made sense to include it as part of the workspaceContext. I'm open to moving it to a new API if you have a strong preference.

Reply:
Thanks @floralan! Yeah Let's go ahead and make that API update.  It will ensure we have access to any configuration without having to update the workspace context 👍